src/components/TeamMember.tsx

```diff
@@ -18,20 +18,43 @@ const TeamMemberContainer = styled(Flex)`
   }
 `;
 
+const pulse = keyframes`
+  0% { background-color: ${color("black10")}; }
+  50% { background-color: ${color("black5")}; }
+  100% { background-color: ${color("black10")}; }
+`;
+
 const AvatarContainer = styled(Box)`
   flex-shrink: 0;
   position: relative;
+  width: 100px;
+  height: 100px;
+  background-color: ${color("black10")};
+  animation: ${pulse} 2s ease-in-out infinite;
+  border-radius: 50%;
+`;
```

Comment:
Had to essentially rip this from palette. We need to export that animation somewhere if we're not already.